draft-ietf-scitt-architecture.md

```diff
@@ -677,165 +555,74 @@ Signed Statements may be registered by a different party than their Issuer.
 The Transparency Service MAY verify the Statement payload format, content and other optional properties.
 1. **Apply Registration Policy:** The Transparency Service MUST check the attributes required by a policy are present in the protected headers.
   Custom Signed Statements are evaluated given the current Transparency Service state and the entire Envelope, and may use information contained in the attributes of named policies.
-1. **Register the Signed Statement** to the append-only log
-1. **Return the Transparent Statement**, which includes the Receipt
-  Details about generating Receipts are described in {{Receipt}}.
+1. **Register the Signed Statement** to the append-only log.
+1. **Return the Receipt**, which MAY be asynchronous from registration.
+Details about generating Receipts are described in {{Receipt}}.
 
 The last two steps may be shared between a batch of Signed Statements recorded in the Append-only Log.
 
 A Transparency Service MUST ensure that a Signed Statement is registered before releasing its Receipt, so that it can always back up the Receipt by releasing the corresponding entry (the now Transparent Statement) in the Append-only Log.
 Conversely, the Transparency Service MAY re-issue Receipts for the Append-only Log content, for instance after a transient fault during Signed Statement registration.
 
-## Receipts & Transparent Statements  {#Receipt}
+## Transparent Statements {#Receipt}
+
+The client (which is not necessarily the Issuer) that registers a Signed Statement and receives a Receipt can produce a Transparent Statement by adding the Receipt to the Unprotected Header of the Signed Statement.
 
 When a Signed Statement is registered by a Transparency Service a Receipt becomes available.
 When a Receipt is included in a Signed Statement a Transparent Statement is produced.
 
 Receipts are based on Signed Inclusion Proofs as described in COSE Signed Merkle Tree Proofs ({{-COMETRE}}).
```

Comment:
How does a transparency service signal that it is using a proof format that is not currently registered  in the draft (ie not RFC9162_SHA256)